tests/demo_test.py

```diff
@@ -1,20 +1,63 @@
-import requests
 from utilities.configuration import *
 from data.payloads import *
-from data.payload_data import *
+from data.user import *
+from data.common_headers import *
+from utilities.methods import *
 
-url = get_config()['API']['main_endpoint']
-headers = {"Content-Type": "application/json"}
-create_booking_response = requests.post(url, json=create_booking_payload(
-    first_name, last_name, total_price, deposit_paid, check_in, check_out, additional_needs), headers=headers, )
+s = requests.Session()
 
-response_json = create_booking_response.json()
+"Create booking"
```

Comment:
Przy testach myślałam o pozamykaniu tego również w metody, albo klasę z funkcjami żeby sam test był tylko wywoływaniem funkcji + asercjami

Reply:
myślę, że wprowadzenie pytesta wszystko uporządkuje :) 

wtedy proponowałabym pojedyncze testy tworzące zamówienie i wykonujące dla niego pewną akcję:
- utworzenie zamówienia + sprawdzenie czy występuje na liście wszystkich zamówień
- utworzenie zamówienia + sprawdzenie czy można o nie odpytać po konkretnym ID
- utworzenie zamówienia + edycja zamówienia + sprawdzenie czy jest zaktualizowane na liście wszystkich zamówień
- utworzenie zamówienia + edycja zamówienia + sprawdzenie czy jest zaktualizowane gdy odpytamy o nie po konkretnym ID
- utworzenie zamówienia + częściowa edycja zamówienia + sprawdzenie czy jest zaktualizowane na liście wszystkich zamówień
- utworzenie zamówienia + częściowa edycja zamówienia + sprawdzenie czy jest zaktualizowane gdy odpytamy o nie po konkretnym ID
- utworzenie zamówienia + usunięcie zamówienia + sprawdzenie czy występuje na liście wszystkich zamówień
- utworzenie zamówienia + usunięcie zamówienia + sprawdzenie czy można o nie odpytać po konkretnym ID

Dla akcji wymagających autoryzacji dodałabym jeszcze testy, które próbuję wykonać pewne akcje bez autoryzacji (powinno być to niemożliwe, pewnie leci jakiś 401 :))
Przy częściowej edycji pewnie można poszaleć z treścią zamówienia. 
Możemy jeszcze pogłówkować na spotkaniu

wyszło z tego całkiem sporo testów, więc pewnie można podzielić je na klasy związane z tworzeniem zamówienia, usuwaniem i edycją (pełną i częściową)